lib/ui/bottom-panel.js

```diff
@@ -1,6 +1,7 @@
 'use babel'
 
 const Interact = require('interact.js')
+const Clipboard = require('clipboard')
```

Comment:
This is an electron module https://github.com/atom/electron/blob/master/docs/api/clipboard.md